### DISCUSSION TITLE
Feature: print charts based on DataviewJS

### DISCUSSION THREAD

**Discussion by ghost** [marijnbent/obsidian-print#15] [Ideas]
Firstly, thanks for a plugin.

Secondly, there is a huge problem related to printing that is not covered by any plugin. 
Printing of dynamic content like dataviewjs charts. 

There are a lot of users who use charts and dynamic tables based on js scripts. 
If you'll try to print note with dataviewjs charts with this plugin - the space would be just empty. (i tried)))

If you'll try to export note with charts to pdf - the final page would be with very big badly formatted chart went away from page proportions. 

 Request: 

Consider implementing a feature, that would handle easy printing of charts inserted into Obsidian note based on dataviewjs scripts.

**Comment by marijnbent** [comment]
Thanks for your comment @Altenrion . If there are more users that request it I will consider adding it!
